### PR TITLE
created project's skeleton code base

### LINKED ISSUE #1: Create project skeleton
Create a skeleton for project's code base

### PR DESCRIPTION
**Pull request by vivek-reddy-1234**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,44 @@
 {
-  "name": "holdings",
-  "version": "0.1.0",
+  "name": "photo-gallery",
```
Comment (wissenshakti) [inline package.json]: need to change this to appropriate project name  . @sachinrath1986  how about wissenPortfolioApp ?

File: package.json [hunk 2]
```diff
@@ -34,5 +63,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "devDependencies": {
+    "@capacitor/cli": "3.1.1"
+  },
+  "description": "An Ionic project"
```
Comment (wissenshakti) [inline package.json]: Need to change the description to appropriate one. Ex: A Portfolio application skeleton to get quickly start with portfolio managment application.

File: public/index.html [hunk 3]
```diff
@@ -2,42 +2,30 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <title>Ionic App</title>
```
Comment (wissenshakti) [inline public/index.html]: Change the title to Wissen Portfolio App

**Review by wissenshakti — CHANGES_REQUESTED** [review]
Had some minor comments . @sachinrath1986  @vivek-reddy-1234  Please take a look and we can discuss further to resolve those or not

File: ionic.config.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "photo-gallery",
```
Comment (sachinrath1986) [inline ionic.config.json]: name!

**Comment by wissenshakti** [comment]
Commit message should include id of issue for ex : ' #1 some minor change' . Also commit message should not be vague
ex instead of your last commit ' #1 Address comments to conform with style guide' . Please edit the last commit to include github id

**Comment by sachinrath1986** [comment]
Merging for now. We can revisit the change requested later.

**Pull request merged**